lib/TimezoneGuesser/FindFromMzVersionTimezone.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabre\VObject\TimezoneGuesser;
+
+use DateTimeZone;
+use Sabre\VObject\TimeZoneUtil;
+
+class FindFromMzVersionTimezone implements TimezoneFinder
```

Comment:
Can you add some description in order to explain what this finder is supposed to take care of? 🙏

Reply:
Added, thanks!